WhaleSpotting/react-app/src/styles/PageNav.scss

```diff
@@ -0,0 +1,31 @@
+@import "./constants.scss";
```

Comment:
Are these styles being imported anywhere? AFAICT they should be in the PageNav.tsx (will review them as such)

Reply:
if we remove the importing program does not recognize the constants, build does not work
Put into PageNav.scss as well